CHANGELOG.md

```diff
@@ -6,6 +6,8 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 - added the build info collector
+- go-kit integration
```

Comment:
FYI: We've been placing a link to the PRs on each change entry. 
```
go-kit integration [#6](https://github.com/xmidt-org/touchstone/pull/6)
```

Reply:
That's a good standard.  I'll steal it :)